src/wwElement.vue

```diff
@@ -1,393 +1,461 @@
-    }
-    &.split-4 {
-        background-color: var(--calendar-split-4-bg-color);
-    }
-    &.split-5 {
-        background-color: var(--calendar-split-5-bg-color);
-    }
-    &.split-6 {
-        background-color: var(--calendar-split-6-bg-color);
-    }
-    &.split-7 {
-        background-color: var(--calendar-split-7-bg-color);
-    }
-    &.split-8 {
-        background-color: var(--calendar-split-8-bg-color);
-    }
-    &.split-9 {
-        background-color: var(--calendar-split-9-bg-color);
-    }
-}
-</style>
+<template>
+    <vue-cal
+        :key="content.themeColor + '-' + content.startWeekOnSunday"
+        :disable-views="disabledViews"
+        :active-view="currentView"
+        :hide-view-selector="disabledViews.length >= 4"
+        :hide-weekends="content.hideWeekends"
+        :events="events"
+        :split-days="calendars"
+        :showAllDayEvents="content.showAllDayEvents"
+        :events-count-on-year-view="content.showCountOnYearView"
+        :locale="currentLang"
+        :class="content.themeColor"
+        :style="{ ...customThemeStyle, ...calendarsStyle, ...categoriesStyle }"
+        :time="!content.enableTimelessMode"
+        :timeStep="content.timestep"
+        :timeFrom="content.timeStart * 60"
+        :timeTo="content.timeEnd * 60"
+        :startWeekOnSunday="content.startWeekOnSunday"
+        :hideWeekdays="content.hideWeekdays"
+        :twelveHour="content.twelveHour"
+        :small="content.daySize === 'small'"
+        :xsmall="content.daySize === 'xsmall'"
+        @event-click="handleEventClick"
+        @cell-click="handleCellClick"
+        @view-change="handleViewChange"
+    />
+</template>
+
+<script>
+import VueCal from 'vue-cal';
+import 'vue-cal/dist/vuecal.css';
+import * as locale_en from 'vue-cal/dist/i18n/en.es.js';
+import * as locale_fr from 'vue-cal/dist/i18n/fr.es.js';
+import * as locale_es from 'vue-cal/dist/i18n/es.es.js';
+import * as locale_de from 'vue-cal/dist/i18n/de.es.js';
+
+const locales = {
+    locale_en,
+    locale_fr,
+    locale_es,
+    locale_de,
+};
+
+export default {
+    components: { VueCal },
+    props: {
+        content: { type: Object, required: true },
+        /* wwEditor:start */
+        wwEditorState: { type: Object, required: true },
+        /* wwEditor:end */
+    },
+    emits: ['trigger-event', 'update:content:effect'],
+    data: () => ({
+        currentView: null,
+    }),
+    computed: {
+        /**
+         * Check if the selected language is supported.
+         * If so, use it. Otherwise, fallback to English.
+         *
+         * @returns {string|string}
+         * @see https://github.com/antoniandre/vue-cal/blob/b41fcee7909b11d0ed5234684171ae792312367b/src/documentation/index.vue#LL362C1-L404C2 List of all supported locales
+         */
+        currentLang() {
+            if (typeof this.content.lang === 'object') {
+                // Object provided, use it as-it (allows locale override, custom locale, etc.)
```

Comment:
Are you sure that using an object works ? 
We could not find this in the documentation.

Reply:
@MarcFabre Yes, 100% sure. I tested it. 
It is a recent addition since v3.9.0

![image](https://github.com/weweb-assets/ww-calendar/assets/107790761/7f15a606-ecba-4fab-9631-60420c03e213)


Here is a video:
https://watch.screencastify.com/v/vdYvqoUteVBuw0c0PS7X